### ISSUE TITLE
[BUG]Memory leaks on AUR/Manjaro

### ISSUE THREAD

**Issue by asorel1942** [Lightcord/Lightcord#30]
**Describe the bug**
Lightcord will use progressively more RAM the longer it stays open.

**To Reproduce**
Open Lightcord, keep open for extended period.

**Expected behavior**
For memory usage to stay relatively constant over time.

**Screenshots**
Attached screenshots show the memory used 18:51, 21:05, and 00:52
![Memory usage 18:51](https://user-images.githubusercontent.com/33983895/91257112-0fa65d00-e737-11ea-9a62-7199141a888e.png)
![Memory usage at 21:05](https://user-images.githubusercontent.com/33983895/91257123-159c3e00-e737-11ea-9151-fae231b2e5bf.png)
![Memory usage at 00:52](https://user-images.githubusercontent.com/33983895/91257135-1df47900-e737-11ea-9c80-31bd83d3c391.png)

**Desktop (please complete the following information):**
Stable 65749 (2934e18)
Lightcord 0.1.4 (80b2b93)
BBD 0.3.5 (lightcord)
Host 0.0.306
Linux 64-Bit (5.8.1-3-MANJARO)

**Additional context**

**Comment by Jhackler** [comment]
Getting same behaviour on arch

**Comment by nephitejnf** [comment]
Does downloading the zip and running it manually get the same effect, cause the PKGBuild uses that just.

**Comment by Jhackler** [comment]
I was going to try the zip, however was running light cord as installed as a baseline, it seems to slowly climb to a point then go up and down so may actually be non issue.

**Comment by Jhackler** [comment]
Yeah after days of running it it seems to slowly go up to a point then stop for me its still less than discord I have 32gb of ram though so no difference. might be a system library on manjaro as they hold back arch updates quite a bit.

**Comment by unixkeys** [comment]
I recently switched back to windows and it seems to do the same thing to me, it's probably because @JeanOUINA used react for literally every part of the gui.
I didn't contribute that much on lightcord, I pretty much did the social part of it (eg: growth), but I plan to contribute to it someday, it would suck to not do so

**Comment by JeanOUINA** [comment]
> it's probably because @JeanOUINA used react for literally every part of the gui.
what else would you like me to use ? if using direct DOM manipulation, discord can crash because react does not recognize the dom. 

Anyway, it shouldn't be react, most likely caching. I use a lot of caching in all sorts of components to speed up the render process when rendering multiple time.

**Comment by JeanOUINA** [comment]
missclick

**Comment by unixkeys** [comment]
what dude it's my little brother he took my pc

**Comment by smartfrigde** [comment]
It's possible one of your plugins had memory leak. Some are not optimised or badly coded.
